modules/testing/pages/integration/canceling.adoc

```diff
@@ -0,0 +1,11 @@
+= Canceling Integration pipelineruns for Pull Requests
+
+Pushing a new change to your Pull Request creates a new build pipelineRun, followed by a new snapshot if build succeeded which then triggers integration test pipelineRuns for that snapshot.
+In case you happen to push another change in your Pull Request while integration tests pipelineRuns are still running, the previous test pipelineRuns will be canceled.
+
+== How do I cancel the previous pipelinerun and which will be canceled?
+
+If your integration tests are still running and you push new changes to your Pull Request, once new build pipelinerun succeeds and new snapshot is created,
+integration service will gracefully cancel previous integration test pipelinerun.
```

Comment:
I feel like this is just restating the previous paragraph. Can we combine them into one concise statement? If we just have one paragraph, does it make more sense to integrate this content into another page?

Reply:
Combined those two paragraphs into one. I think the logic was preserved.

Reply:
I think that this page is still confusing. When I see this topic, I feel like it will describe to me how _I_ can cancel ITS runs. But I think this is just stating how ITS self-cancels outdated PLRs? 

If so, can we combine this with a different page?

Reply:
Would it help if we expanded the document to also describe how to manually cancel an ongoing ITS run from the Konflux UI?
It would be basically mentioning that you can press the `cancel` option from the dropdown menu next to the pipelineRun in the UI:
![image](https://github.com/user-attachments/assets/39c1a695-a61d-4969-b725-fe13d1e933a0)

Reply:
Yeah, I think that would make it better.